controllers/api.controller.js

```diff
@@ -0,0 +1,57 @@
+const Url = require('../models/url.model');
```

Comment:
Give your variable names a meaning - in this case, "Url" for your model rather than "Model"